.changeset/few-melons-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/button': minor
+---
+
+Prevents `onSubmit` handler from firing when `disabled="true"` or `isLoading="true"`
```

Comment:
We should update this changelog comment to be more reflective of the new change

Reply:
Updated, lmk what you think.